src/Engine/Renderer/RenderTechnique/ShaderProgram.hpp

```diff
@@ -63,9 +63,25 @@ class RA_ENGINE_API ShaderProgram final {
 
     void setUniform( const char* name, Texture* tex, int texUnit ) const;
 
+    //! use automatic texture unit computation
+    //! if you want to send a particular texture unit, use setUniform.
+    //! It binds tex on an "arbitrary" tex unit.
+    //! @warning, call a std::map::find (in O(log(active tex unit in the shader)))
+    void setUniformTexture( const char* texName, Texture* tex ) const;

```

Comment:
Use Radium recommended style for long documentation (/** ... */). Document the parameters. Is ``texName`` the same as ``tex->getName()`` ? Might it be ``samplerName`` instead ?

Reply:
well, I was not aware that long doc shoud use /** */, but it do not ease the quick comment of large block